tests/cpp/include/test_mkldnn.h

```diff
@@ -330,7 +330,7 @@ inline void PrintVerifyMsg(const NDArrayAttrs &arr1, const NDArrayAttrs &arr2) {
  */
 inline std::vector<NDArrayAttrs> GetTestInputArrays(
     int types = ArrayTypes::All, bool rand = false,
-    std::vector<float> scale = {1}, bool spatial_data_format = false) {
+    std::vector<float> scale = {1}, bool spatial_data_format = false, int max = 50) {
```

Comment:
Is the new parameter for better usability?

Reply:
See https://github.com/apache/incubator-mxnet/pull/17318#discussion_r379465354.